app/models/ability.rb

```diff
@@ -84,6 +85,12 @@ def can_leave_event(user)
       end
     end
 
+    def can_update_gameday(user)
+      can :update, Gameday do |gameday|
+        not (gameday.event.organizers & user.organizers).empty?
```

Comment:
Why `user.organizers`?

Reply:
because organizer is a join between even and user. It belongs to one user and one event. And because there is no `has_many: users, through: organizers` in Event and no `has_many: events, through: organizers` in User. I have to compare them like this.

Reply:
Mabye for clarification: `&` is the cut volume of two arrays.